docs/tutorials/01-intro.md

```diff
@@ -127,6 +127,7 @@ With `buildah` files can be copied into the new image.  We can then use `buildah
     # dnf -y install podman
     # buildah copy $newcontainer ./runecho.sh /usr/bin
     # buildah config --cmd /usr/bin/runecho.sh $newcontainer
+    # buildah commit $newcontainer $newcontainer
```

Comment:
It would be clearer here to declare an image name perhaps newimage, and then later have podman run newimage.

Reply:
@rhatdan , changed the image name from `$newcontainer` to `newimage`.